phpdoc.dist.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<phpdoc>
+	<title>WordPress.com VIP – VIP Go Function Documentation</title>
+	<parser>
+		<!-- This places the cache dirs in the root dir `/phpdoc/` for your site -->
+		<target>../../phpdoc/</target>
+		<encoding>utf8</encoding>
+		<markers>
+			<item>TODO</item>
+			<item>FIXME</item>
+		</markers>
+		<visibility></visibility>
+	</parser>
+	<transformer>
+		<!-- This places the docs in the root dir `/phpdoc/` for your site -->
+		<target>../../phpdoc/</target>
```

Comment:
> Keeping them in the same directory probably avoids confusion.

I was trying to keep the generated docs out of the MU plugins directory, out of concern that the nested Git clones (having the `gh-pages` branch checked out inside the other branch) would create some additional unanticipated complexity. Seems fine though.